docs/smart-agent-quickstart.md

```diff
@@ -1,132 +1,298 @@
-Ensure that the folowing dependencies are installed:
-- [.Net Framework 3.5](https://docs.microsoft.com/en-us/dotnet/framework/install/dotnet-35-windows-10) (Windows 8+)
-- [Visual C++ Compiler for Python 2.7](https://www.microsoft.com/EN-US/DOWNLOAD/DETAILS.ASPX?ID=44266)
+#### RPM Package
 
-Once the dependencies have been installed, use the following powershell script
-to install the agent.  The agent will be installed as a Windows service and will
-log to the Windows Event Log.
+We provide a RHEL/RPM package repository that you can use with the following commands:
 
 ```sh
-& {Set-ExecutionPolicy Bypass -Scope Process -Force; $script = ((New-Object System.Net.WebClient).DownloadString('https://dl.signalfx.com/signalfx-agent.ps1')); $params = @{access_token = "YOUR_SIGNALFX_API_TOKEN"; ingest_url = "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com"; api_url = "https://api.YOUR_SIGNALFX_REALM.signalfx.com"}; Invoke-Command -ScriptBlock ([scriptblock]::Create(". {$script} $(&{$args} @params)"))}
+cat <<EOH > /etc/yum.repos.d/signalfx-agent.repo
+[signalfx-agent]
+name=SignalFx Agent Repository
+baseurl=https://dl.signalfx.com/rpms/signalfx-agent/final
+gpgcheck=1
+gpgkey=https://dl.signalfx.com/yum-rpm.key
+enabled=1
+EOH
+
+yum install -y signalfx-agent
+
 ```
 
-Your SignalFx API Token can be obtained from the Organization->Access Token tab in [SignalFx](https://app.signalfx.com).
 
-More detailed installation steps to install via a config management tool or using a containerized agent can be found [here](../README.md#installation).
+## Step 5. Add a New Monitor 
 
-#### Step 2: Configuration
+You may also want to add and configure various monitors for your environment. 
 
-The default configuration file should be located at `/etc/signalfx/agent.yaml` on Linux
-and `\ProgramData\SignalFxAgent\agent.yaml` on Windows.
-Also, by default, the file containing your SignalFx API token should be located at
-`/etc/signalfx/token` on Linux and `\ProgramData\SignalFxAgent\token` on Windows.
+* To [add a new monitor](#Monitors).
+* For [Common configuration options](https://docs.signalfx.com/en/latest/integrations/agent/monitor-config.html).
+* For [Windows monitor configurations](https://docs.signalfx.com/en/latest/integrations/agent/windows.html).
+* For [Linux monitor configurations](https://docs.signalfx.com/en/latest/integrations/agent/monitor-config.html)
 
-In the referenced example agent.yaml configuration files below, the default
-location for the token file is used.
 
-- [Linux Default Configuration File](https://github.com/signalfx/signalfx-agent/blob/master/packaging/etc/agent.yaml)
+### Configure optional metrics for your monitors. 
 
-- [Windows Default Configuration File](https://github.com/signalfx/signalfx-agent/blob/master/packaging/win/agent.yaml)
+See individual monitor pages for your monitor from the lists in Step 5 or see the [Integrations tab](https://app.signalfx.com/#/integrations) in the SignalFx application for Monitor Services.
```

Comment:
Instead of "step 5" xref to whatever heading that corresponds to.

Also, any links to app.signalfx.com need to include the realm info.

Reply:
I don't understand what you mean by links to app.signalfx.com "need to include realm info".

Reply:
Removed the Step 5 reference. Links to the Monitor lists are directly above.

Reply:
Oops. Now I understand what you mean. I fixed it. Thanks for your thoughtful review.